pyphs/numerics/cpp/tools.py

```diff
@@ -47,7 +47,7 @@ def main_path(simu):
 
 
 def dereference(method):
-    return method.args() + [k for k in method.subscpp]
+    return method.args() + method.Vector(*[k for k in method.subscpp])
```

Comment:
Which should be the type of the output ?
depends on the type of method.args which I could not infer...
If method.args() is a mere list/tuple (or another class with a __add__ method), output will be of the same type